### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by BrainonSilicon**
Added information about templating the repo in the readme

### CODE REVIEW

**Comment by BrainonSilicon** [comment]
When you have reviewed the PR - can you also tag all-contributors to add yourselves with doc, talk, mentoring. (and anything else you want)

**Review by cassgvp — CHANGES_REQUESTED** [review]
line 23, missing space 
> remeber to selectthe `Template` button"

File: README.md [hunk 1]
```diff
@@ -6,19 +6,21 @@
 This repository has been set up to accompany the Introduction to GitHub workshop. 
 
 
-## About this Repository
+## About the README
 
-This repository takes inspiration and guidance provided in *[The Turing Way](https://the-turing-way.netlify.app/welcome)* handbook to data science on how to set up and structure a repository to support collaboration in data science projects. It includes files and directories that are recommended for enabling collaboration in a project, as well as sharing of research outputs.
+In an online repository, such as GitHub, the project overview page is named ‘README’ which is equivalent to the main page of a website.
+README page should describe the project -- what is the purpose of the project, who is involved, how to collaborate and where to find key resources.
```
Comment (eirini-zormpa) [inline README.md:12]:
```suggestion
The README page should describe the project -- what the purpose of the project is, who is involved, how to collaborate and where to find key resources.
```

**Comment by malvikasharan** [comment]
@all-contributors please add @cassgvp for doc, talk and mentoring

File: README.md [hunk 2]
```diff
@@ -6,19 +6,21 @@
...
+This repository takes inspiration and guidance provided in *[The Turing Way](https://the-turing-way.netlify.app/welcome)* handbook to data science on how to set up and structure a repository to support collaboration in data science projects. It includes files and directories that are recommended for enabling collaboration in a project, as well as sharing of research outputs.
 
-To learn more about how to create a README.md file, please read the [Landing Page - README File](https://the-turing-way.netlify.app/project-design/project-repo/project-repo-readme.html) chapter in The Turing Way Guide for Project Design.
+When reusing this repository, please update information on this README page with information about your project. 
+
+When it comes to using this template repo for your project work - remeber to selectthe `Template` button and **Create a New Repository**. All the files will be copied across and set up in a new repo for your project, You will need to decide who in your group will do this step to become the "owner" of the repository and then you can submit work via Branches, Forks, and Pull Requests using the skills you have learnt in this workshop
```
Comment (eirini-zormpa) [inline README.md:23]:
```suggestion
When it comes to using this template repo for your project work - remeber to select the `Template` button and **Create a New Repository**. All the files will be copied across and set up in a new repo for your project, You will need to decide who in your group will do this step to become the "owner" of the repository and then you can submit work via Branches, Forks, and Pull Requests using the skills you have learnt in this workshop
```

Comment (eirini-zormpa) [inline README.md:23]:
```suggestion
When it comes to using this template repo for your project work - remeber to selectthe `Template` button and **Create a New Repository**. All the files will be copied across and set up in a new repo for your project. You will need to decide who in your group will do this step to become the "owner" of the repository and then you can submit work via Branches, Forks, and Pull Requests using the skills you have learnt in this workshop
```

**Review by eirini-zormpa — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (eirini-zormpa) [reply README.md:23]: @all-contributors please add @eirini-zormpa for doc and mentoring

**Comment by BrainonSilicon** [comment]
> line 23, missing space
> 
> > remeber to selectthe `Template` button"

@cassgvp fixed by EZ's patch 🎉

**Review by cassgvp — APPROVED** [review]

**Comment by BrainonSilicon** [comment]
@all-contributors please add @malvikasharan for review, mentoring, content, and tools.

**Comment by BrainonSilicon** [comment]
@all-contributors please add @EKaroune for talk, review, and mentoring.

**Comment by BrainonSilicon** [comment]
Thanks everyone!!

**Pull request merged**